qcodes/dataset/plotting.py

```diff
@@ -177,7 +184,8 @@ def plot_by_id(run_id: int,
             if rescale_axes:
                 _rescale_ticks_and_units(ax, data, colorbar)
             if smart_colorscale:
```

Comment:
could we add a info log message here with the limits used and how much is clipped